UI/src/view/linear_asset/linearAssetForm.js

```diff
@@ -107,11 +107,11 @@
 
     var limitValueButtons = function() {
       var separateValueElement =
-        formElements.singleValueElement(selectedLinearAsset.getValue(), "a") +
-        formElements.singleValueElement(selectedLinearAsset.getValue(), "b");
+        formElements.singleValueElement(selectedLinearAsset.getValue(), "a", selectedLinearAsset.getId()) +
+        formElements.singleValueElement(selectedLinearAsset.getValue(), "b", selectedLinearAsset.getId());
       var valueElements = selectedLinearAsset.isSplitOrSeparated() ?
         separateValueElement :
-        formElements.singleValueElement(selectedLinearAsset.getValue());
+        formElements.singleValueElement(selectedLinearAsset.getValue(), undefined, selectedLinearAsset.getId());
```

Comment:
do you need to pass as side code undifined? You can just do it with an empty string

Reply:
yes, the check is if (_.isUndefined(sideCode)) {}else..

Reply:
It could be changed to _isEmpty